beginend.el

```diff
@@ -225,16 +225,41 @@ BEGIN-BODY and END-BODY are two `progn' expressions passed to respectively
   (progn
     (prodigy-last)))
 
+(add-to-list 'beginend-modes (cons 'mu4e-view-mode-hook #'beginend-message-mode))
```

Comment:
I would move these values directly into the `defvar` default value.  The reason is that if alter the code is somehow reordered Emacs might complain about undefined variables.

Also people seem to not like top-level forms which aren't `defXYZ` (I don't have a strong opinion about this though).